objects/toPairs.js

```diff
@@ -0,0 +1,13 @@
+const toPairs = (obj) => {
+    const result = [];
+  
+    for (const key in obj) {
+      if (Object.prototype.hasOwnProperty.call(obj, key)) {
```

Comment:
I suggest to use `Object.hasOwn` it's a new one. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn